metricbeat/etc/beat.yml

```diff
@@ -2,9 +2,11 @@ metricbeat:
   modules:
 
     # Apache Module
+    # Server-status page (/server-status) is configurable string in apache and
+    # must be defined by user
     - module: apache
       metricsets: ["status"]
-      hosts: ["http://127.0.0.1/"]
+      hosts: ["http://127.0.0.1/server-status"]
```

Comment:
Perhaps a better option could be to introduce an optional config for the server-status path. The reasoning is that the host url could be also used for other metricsets which are not necessarly coming from server-status. 

So the config would be for example:
```
metricbeat:
  modules:

    # Apache Module
    # Server-status page (/server-status) is configurable string in apache and
    # must be defined by user
    - module: apache
      metricsets: ["status"]
      hosts: ["http://127.0.0.1/"]
      # Setting server status path.
      #servier_status_path: "server-status"
      period: 1s
      enabled: true
```